src/api/java/com/minecolonies/api/tileentities/TileEntityRack.java

```diff
@@ -340,15 +340,7 @@ public void read(final CompoundNBT compound)
             size = compound.getInt(TAG_SIZE);
             if (size > 0)
             {
-                inventory = new RackInventory(DEFAULT_SIZE + size * SLOT_PER_LINE)
-                {
-                    @Override
-                    protected void onContentsChanged(final int slot)
-                    {
-                        updateItemStorage();
-                        super.onContentsChanged(slot);
-                    }
-                };
+                inventory = new RackInventory(DEFAULT_SIZE + size * SLOT_PER_LINE);
```

Comment:
our RackInventory class already has that overwrite, onContentsChanged was overwritten twice on accident

Reply:
ohh

Reply:
Was just checking that, glad to hear